integreat_cms/cms/constants/recurrence.py

```diff
@@ -22,11 +22,14 @@
 
 #: Events that are recurring, i.e. take place more than once
 RECURRING: Final = 1
-#: Events that are not recurring, i.e. take place only once
-NOT_RECURRING: Final = 2
+#: Events that are not recurring and only take place on one day
+SINGLE_DAY: Final = 2
```

Comment:
Nit: currently "One-time" is used for this kind of event. Probably we should align with the existing naming schema or change the existing one.

Reply:
Good point. I changed them now :)